coderd/members_test.go

```diff
@@ -13,6 +13,43 @@ import (
 	"github.com/coder/coder/v2/testutil"
 )
 
+func TestAddMember(t *testing.T) {
+	t.Parallel()
+
+	t.Run("OK", func(t *testing.T) {
```

Comment:
I think testing to error cases like missing user and org 404, etc for posterity could be good as well